### PR TITLE
Move documentation into docs

### PR DESCRIPTION
**Pull request by mathias-arm**
Change extension from .markdown to .md

### CODE REVIEW

**Comment by egrimley-arm** [comment]
It looks like you need to change `proxy_cleanup.sh` to `sdk/proxy_cleanup.sh` in https://github.com/veracruz-project/veracruz/blob/785c9746c52a4994df6f0530a8e8fce99edea3a8/.github/workflows/main.yml#L87

**Comment by egrimley-arm** [comment]
Maybe do something like:
```
perl -i -pe 's/(LICENSING|LICENSE_MIT)\.markdown/LICENSE.md/g;' `find githooks/pre-commit sdk/veracruz-mcu-client sdk/wasm-checker/src/main.cc -type f`
```

**Comment by mathias-arm** [comment]
> Maybe do something like:

You are right, I missed the files in `sdk` and `githooks`. I will wait for the CI run to finish before I push new changes.

**Comment by egrimley-arm** [comment]
More files to rename:
```
crates/examples/rust-examples/huffman-encoding/README.markdown
crates/examples/rust-examples/prime-numbers-generator/README.markdown
crates/examples/rust-examples/string-search/README.markdown
crates/execution-engine/README.markdown
sdk/veracruz-mcu-client/MCU_CLIENT_INSTRUCTIONS.markdown
```

**Comment by mathias-arm** [comment]
> More files to rename

Also added to latest push.

File: crates/examples/data-generators/image-processing-generator/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+We use the image [Veracruz Puerto - Vista desde el Hotel Emporio](veracruz.jpg) by Eduardo Pavon as a test dataset for some of our examples.
+This image is licensed under the [Creative Commons Attribution Share Alike 2.0 Generic (CC-BY-SA 2.0) license](LICENSE_CC-BY-SA-2.0.md).
```
Comment (egrimley-arm) [inline crates/examples/data-generators/image-processing-generator/README.md:2]: A newline at the end of this file would be good.

File: README.md [hunk 2]
```diff
@@ -65,6 +67,4 @@ If you use Veracruz or otherwise wish to discuss the project, please cite the pr
 
 ## License
 
-This project's codebase is licensed under the [MIT license](LICENSE_MIT.markdown).
-
-We use the image [Veracruz Puerto - Vista desde el Hotel Emporio](examples/data-generators/image-processing-generator/veracruz.jpg) by Eduardo Pavon as a test dataset for some of our examples.  This image is licensed under the [Creative Commons Attribution Share Alike 2.0 Generic (CC-BY-SA 2.0) license](LICENSE_CC-BY-SA-2.0.markdown).
+This project's codebase is licensed under the [MIT license](LICENSE.md).
```
Comment (egrimley-arm) [inline README.md:70]: Perhaps it would be good to write: Except where stated otherwise, this project's codebase is ...

Comment (mathias-arm) [reply README.md:70]: Since the image is not part of the *codebase*, I thought the statement was still true.

Comment (egrimley-arm) [reply README.md:70]: You're probably right.

Comment (mathias-arm) [reply README.md]: It does not hurt to add it.

File: crates/examples/data-generators/image-processing-generator/README.md (hunk 1 above)
Comment (mathias-arm) [reply crates/examples/data-generators/image-processing-generator/README.md]: Done

**Comment by mathias-arm** [comment]
Latest push should address all comments

**Review by egrimley-arm — APPROVED** [review]

**Pull request merged**